packages/iTwinUI-react/src/core/Dialog/DialogWrapper.tsx

```diff
@@ -0,0 +1,34 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import React from 'react';
+import cx from 'classnames';
+import { useTheme } from '../utils';
+import '@itwin/itwinui-css/css/dialog.css';
+
+export type DialogWrapperProps = React.ComponentPropsWithRef<'div'>;
+
+/**
+ * Dialog wrapper used for draggable dialogs in order to make them draggable relatively to this container.
+ * @example
+ * <DialogWrapper>
+ *   <Dialog>
+ *    ...
+ *  </Dialog>
+ * </DialogWrapper>
+ */
+export const DialogWrapper = React.forwardRef<
```

Comment:
It is not `Dialog.Wrapper` because I wanted to have only dialog inner sub-components under `Dialog.*`.